mixins/localize-dynamic-mixin.js

```diff
@@ -0,0 +1,57 @@
+import { getLocalizeOverrideResources } from '../helpers/getLocalizeResources.js';
+import { LocalizeMixin } from './localize-mixin.js';
+
+const fallbackLang = 'en';
+const defaultExportName = 'default';
+
+function warnMissingFiles(failures, resolvedLang) {
+	const { path } = failures[0].err.message.match(/(?:https?:\/\/.*?\/)(?<path>.*)\/.*\./).groups;
+	const langs = failures.map(f => f.lang).join('","');
+	console.warn(
+		'WARNING: Unacceptable in production. Consider Manually Retrieved Resources method.\n\n' +
+		`Failed to load translation resources for languages "${langs}" from ${path}.`,
+		`Falling back to "${resolvedLang}". `);
+}
+
+export const LocalizeDynamicMixin = superclass => class extends LocalizeMixin(superclass) {
+
+	static get config() {
```

Comment:
Wondering if we should call this something like `localizeConfig` to avoid possible collisions with other mixins or properties/methods on the element?